### PR TITLE
disable notifications requested in https://github.com/DOAJ/doajPM/iss…

### PR DESCRIPTION
**Pull request by richard-jones**
…ues/3974

* Issue: https://github.com/DOAJ/doajPM/issues/3974
---

# Remove some notifications

Some notifications have been removed.  The code and tests remain, they have just been moved into an inactive state.

This PR...
- [ ] has scripts to run
- [ ] has migrations to run
- [ ] adds new infrastructure
- [ ] changes the CI pipeline
- [ ] affects the public site
- [x] affects the editorial area
- [x] affects the publisher area
- [ ] affects the monitoring

## Developer Checklist

*Developers should review and confirm each of these items before requesting review*

* [x] Code meets acceptance criteria from issue
* [x] Unit tests are written and all pass
* [x] User Test Scripts (if required) are written and have been run through
* [x] Project's coding standards are met
    - No deprecated methods are used
    - No magic strings/numbers - all strings are in `constants` or `messages` files
    - ES queries are wrapped in a Query object rather than inlined in the code
    - Where possible our common library functions have been used (e.g. dates manipulated via `dates`)
    - Cleaned up commented out code, etc
    - Urls are constructed with `url_for` not hard-coded
* ~[ ] Code documentation and related non-code documentation has all been updated~
    - ~Core model documentation has been added to if needed: https://docs.google.com/spreadsheets/d/1lun2S9vwGbyfy3WjIjgXBm05D-3wWDZ4bp8xiIYfImM/edit~
    - ~Events and consumers documentation has been added if needed: https://docs.google.com/spreadsheets/d/1oIeG5vg-blm2MZCE-7YhwulUlSz6TOUeY8jAftdP9JE/edit~
* ~[ ] Migration has been created and tested~
* [x] There is a recent merge from `develop`

## Reviewer Checklist

*Reviewers should review and confirm each of these items before approval*
*If there are multiple reviewers, this section should be duplicated for each reviewer*

* [ ] Code meets acceptance criteria from issue
* [ ] Unit tests are written and all pass
* [ ] User Test Scripts (if required) are written and have been run through
* [ ] Project's coding standards are met
    - No deprecated methods are used
    - No magic strings/numbers - all strings are in `constants` or `messages` files
    - ES queries are wrapped in a Query object rather than inlined in the code
    - Where possible our common library functions have been used (e.g. dates manipulated via `dates`)
    - Cleaned up commented out code, etc
    - Urls are constructed with `url_for` not hard-coded
* [ ] Code documentation and related non-code documentation has all been updated
    - Core model documentation has been added to if needed: https://docs.google.com/spreadsheets/d/1lun2S9vwGbyfy3WjIjgXBm05D-3wWDZ4bp8xiIYfImM/edit
    - Events and consumers documentation has been added if needed: https://docs.google.com/spreadsheets/d/1oIeG5vg-blm2MZCE-7YhwulUlSz6TOUeY8jAftdP9JE/edit
* [ ] Migation has been created and tested
* [ ] There is a recent merge from `develop`

## Testing

Attempt to trigger one of the removed notifications, for example:

* Find an open application with no editor assigned, go into the application and assign an associate editor.  A notification should be sent to the editor, but not to the publisher

## Deployment

No deployment considerations

### CODE REVIEW

File: portality/bll/services/events.py [hunk 1]
```diff
@@ -26,32 +26,32 @@
 
 
 class EventsService(object):
-    # disabled events - to enable move the event to EVENT_CONSUMENRS array
+    # disabled events - to enable move the event to EVENT_CONSUMERS array
     DISABLED_EVENTS = [
```
Comment (Steven-Eardley) [inline portality/bll/services/events.py:30]: Disabling events will require an update to the tests in https://github.com/DOAJ/doaj/blob/develop/doajtest/unit/application_processors/test_application_processor_emails.py

**Review by Steven-Eardley — CHANGES_REQUESTED** [review]
Approach is great, but there's some tests to update before this is ready

File: portality/bll/services/events.py (hunk 1 above)
Comment (richard-jones) [reply portality/bll/services/events.py:30]: fixed

**Review by Steven-Eardley — APPROVED** [review]
Super. This can go out this week with t'others

**Pull request closed without merging**